### ISSUE TITLE
Could not read all champion data from memory

### ISSUE THREAD

**Issue by Xahrie** [floh22/LeagueBroadcastHub#172]
I have this issue running on the current patch (12.12.1).

![image](https://user-images.githubusercontent.com/39407849/175932298-a89865e9-7cdd-400b-86ac-fa0a9d4795c4.png)

**Comment by daisukedd** [comment]
This problem happens sometimes and it is possible to continue using the program without problems, just ignore it, in some cases just restarting the software solves it. But when did that happen? when opening? During Replay? Did you pause the game or go back in time?

**Comment by Xahrie** [comment]
List of things that are not working:
- Team gold stays at 2,500g
- CS / min graph shows incorrect data (i.e. Kayn as 2nd place - but he had the 3rd lowest cs)
- Gold / min shows nothing or the cs graph (same with XP)
- Item purchases and levelups are not displayed
- when elder or nashor are up the icon is not displayed but after it was killed the icon will show up
- upon disabling "show powerplay" the icon will go away and shows the remaining time of the buff but the timer doesn't count backwards and gold stays 0
- when the buff is expired the icon of elder/nash will go away and it shows the time of the powerplay "0:00" with 0 Gold
- the gold graph is empty

It occurred after spectate start at 0:00 (game start). Even if i close it and open it once again. Same with live specting and vod. I didn't paused. 

I tried to disable the "CheckForOffsets", changed the patch to 12.11.1 and manually copied the farsight.json of 12.11.1 and duplicated the cache folder with the name "12.11.1" and changed the patch from "latest" to "12.11.1".

Then I changed it back and renamed the game version of farsight.json to 12.12.1.

Every attempt ended within the same result. As explained above.

The new champion "Bel'Veth" was never involved.
